### PR TITLE
[WEB-251] Allow fuzzy searching in the utility multi-select

### PR DESCRIPTION
**Pull request by conwaydev**
## Details of Change
 
* Updates build process to use webpack instead of browserfy to allow us to write es6 code
* Updates the select-multi logic to allow for "fuzzy searching"

## Relevant Links
 - [WEB-251](https://sprout.atlassian.net/browse/WEB-251)
- [Test Site](https://justin-test.stagely.sproutsocial.com/?tailor=1)

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        admin: './assets/js/src/admin.js',
+        tinymce: './assets/js/src/tinymce.js',
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader",
```
Comment (ericcecchi) [inline webpack.common.js]: Does this need a Babel config?

**Comment by jritterbush** [comment]
@conwaydev the utilities aren't loading for me on your test box

**Comment by conwaydev** [comment]
@jritterbush redeployed: https://cl.ly/0b4cb02bc7b2

File: webpack.common.js (hunk 1 above)
Comment (conwaydev) [reply webpack.common.js]: if you pull the PR you see the minified JS files aren't different which leads me to believe that I don't, any rules we can put inside this config

Comment (ericcecchi) [reply webpack.common.js]: The minified sidebar JS committed here is different in that it now contains `const`s and arrow functions. If we use any more ES6+ syntax/functions, those won't be transpiled either.

**Comment by ericcecchi** [comment]
Something seems off with the encoding on your test site: https://cl.ly/aa693956e012

Seeing that in Firefox.

**Comment by conwaydev** [comment]
Gonna pull out the webpack changes and separate them in a different PR, i think thats screwing with it

**Review by ericcecchi — APPROVED** [review]

**Review by jritterbush — APPROVED** [review]

**Pull request merged**